public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,142 @@
+block includes
+  include ../_util-fns
+
+:marked
+  # Trying Angular
+
+  Live coding [Quickstart](../quickstart.html) in the browser is a great way to explore Angular.
+  Links on almost every documentation page will open completed samples in the browser where you can
+  play with the code, share your changes with friends, and download and run the code on your own machine.
+
+  As much fun as this is
+  * you can't ship your app in plunker
+  * you aren't always online when writing code
+  * transpiling TypeScript in the browser is slow
+  * the type support, refactoring, and code completion only work in your local IDE
+ 
+  # Setup a local development environment
+
+  Fortunately, setting up a new project in your local development environment is quick and easy
+  with the [**QuickStart seed**](https://github.com/angular/quickstart "Install the github QuickStart repo"),
+  a repository on github. 
+
+  Make sure you have [node and npm installed](#install-node "What if you don't have node?").
+  Then ...
+  1. [clone](#clone) or [download](#download) the **QuickStart seed repository**.
+  1. [install the `npm` packages](#install-node "What if you don't have node?")
+  1. run `npm start` to launch the sample application
+
+a#clone
+:marked
+  ### Clone
+
+  Perform the _clone-to-launch_ steps with these terminal commands.
+
+code-example(language="sh" class="code-shell").
+  git clone https://github.com/angular/quickstart.git my-project
+  cd my-project
+  npm install
+  npm start
+
+a#download
+:marked
+  ### Download
+  <a href="https://github.com/angular/quickstart/archive/master.zip" title="Download the QuickStart seed repository">Download the QuickStart seed</a>
+  and unzip it into your project folder. Then perform the remaining steps with these terminal commands.
+
+code-example(language="sh" class="code-shell").
+  cd my-project
+  npm install
+  npm start
+
+.l-main-section
+:marked
+  ## What's different about the QuickStart Seed?
+
+  The QuickStart seed contains the same application as the <live-example><i>live code</i> QuickStart</live-example>.

```

Comment:
You'll need a `name` property if you want this to link to the QS plunker. Also, why not write it as:
```
<live-example name="quickstart">QuickStart live example</live-example>
```